### PR TITLE
Delete some dead code.

### PR DESCRIPTION
**Pull request by itamarst**
Fixes https://tahoe-lafs.org/trac/tahoe-lafs/ticket/3340

### CODE REVIEW

File: src/allmydata/util/fileutil.py [hunk 1]
```diff
@@ -538,60 +538,6 @@ def get_available_space(whichdir, reserved_space):
...
-            raise WinError(get_last_error())
-
-        CloseHandle(hVolume)
-else:
-    def flush_volume(path):
-        # use sync()?
-        pass
-
-
 class ConflictError(Exception):
```
Comment (meejah) [inline src/allmydata/util/fileutil.py:541]: Is all this really unused, or did it just look that way to a tool that was run on linux (instead of windows)?

Comment (itamarst) [reply src/allmydata/util/fileutil.py:541]: I grepped all functions I deleted, couldn't find this one. I'll check again.

Comment (itamarst) [reply src/allmydata/util/fileutil.py:541]: Yep can't find it.

**Comment by itamarst** [comment]
I wonder if I screwed up the grepping, given the failures...

File: src/allmydata/util/eliotutil.py [hunk 2]
```diff
@@ -246,53 +214,6 @@ def parse(self, description):
...
-            def get_file():
-                path.parent().makedirs(ignoreExistingDirectory=True)
-                return LogFile(
-                    path.basename(),
-                    path.dirname(),
-                    rotateLength=rotate_length,
-                    maxRotatedFiles=max_rotated_files,
-                )
-        return lambda reactor: FileDestination(get_file())
-
```
Comment (exarkun) [inline src/allmydata/util/eliotutil.py:295]: This deletion looks wrong to me.  Was it automatically determined by a tool that didn't notice the `getattr(self, u"_parse_{}".format(kind))` above?

File: src/allmydata/util/eliotutil.py [hunk 3]
```diff
@@ -75,23 +60,6 @@
...
-    return validator
-
-def validateSetMembership(s):
-    """
-    Return an Eliot validator that requires values to be elements of ``s``.
-    """
-    def validator(v):
-        if v not in s:
-            raise ValidationError("{} not in {}".format(v, s))
-    return validator
```
Comment (exarkun) [inline src/allmydata/util/eliotutil.py:94]: Hm.  These were used by magic-folder before it was split out into its own package.  I guess I shouldn't say we should keep them given that they're now unused.  We can always pull them back out of git history if we decide we want them ...

The larger story about logging in Tahoe-LAFS is that it leans heavily on Twisted and Foolscap logging and I'd prefer it largely switch to Eliot, though, so I suspect we will want these back.

File: src/allmydata/util/fileutil.py (hunk 1 above)
Comment (exarkun) [reply src/allmydata/util/fileutil.py:541]: for what it's worth, it looks like it was introduced for the magic-folder predecessor, "drop upload".

File: src/allmydata/util/eliotutil.py (hunk 2 above)
Comment (itamarst) [reply src/allmydata/util/eliotutil.py:295]: My fault, I just searched with grep manually.

File: src/allmydata/util/eliotutil.py (hunk 3 above)
Comment (itamarst) [reply src/allmydata/util/eliotutil.py:94]: I can keep the Eliot stuff it you think it's likely to be used eventually.

**Comment by itamarst** [comment]
@exarkun how's this?

**Pull request merged**